modules/porous_flow/doc/content/modules/porous_flow/ates.md

```diff
@@ -0,0 +1,86 @@
+
+- Water is produced from an aquifer;
+- The thermal energy from some external source (e.g. excess renewable energy or industrial waste heat) is transferred to the water;
+- The hot water is re-injected into the aquifer, where it is stored until it is needed;
+- When needed, the hot water is produced, and the energy extracted.
+
+This process can be reversed to enable cooling. The duration of an ATES cycle can range from hours to months, depending on the intended use of the energy; for example, storing excess solar energy during the day and extracting it for use at night (daily cycle); or, the very common case of storing excess heat energy in the warmer months and extracting it for use in the colder months (annual cycle). As such, ATES systems can be used to address energy storage challenges that arise from the intermittent nature of renewable energy and other sources.
+
+There are many ATES systems in operation currently.  Their viability depends crucially on the recovery efficiency, $R$, which is the ratio of heat energy extracted to heat energy injected, because it is impossible to extract all of the injected heat from an ATES system due to heat losses caused by conduction and convection.  $R$ is
+
+\begin{equation}
+R = \frac{\overline{h}_{\mathrm{p}} - h_{\mathrm{amb}}}{h_{\mathrm{i}} - h_{\mathrm{amb}}} 
+      \approx \frac{\overline{T}_{\mathrm{p}} - T_{\mathrm{amb}}}{T_{\mathrm{i}} - T_{\mathrm{amb}}} \ ,
+\end{equation}
+
+where $h$ denotes enthalpy and $T$ temperature.  The subscript "p" indicates "produced" (retrieved from the aquifer), "amb" indicates "ambient", and "i" indicates "injected".  For instance, if the ambient aquifer temperature is $T_{\mathrm{amb}} = 20^{\circ}$C, the injection temperature is $T_{i} = 150^{\circ}$C and $\reff=0.9$, then the average produced temperature is $\overline{T}_{\mathrm{p}} \approx 137^{\circ}$C.
+
+$R$ has been measured at ATES sites, and estimated using numerical modelling.  Numerical models can provide accurate estimates of $R$ as a function of operational parameters, such as the injection temperature and rate, and aquifer parameters such as the permeability, depth.  Such estimates are useful for rapid screening of potential ATES sites, indicating which sites might or might not be viable.
+
+The purpose of this page is to describe a MOOSE model of an ATES system, with the goal of predicting $R$, and present a few results.  For significantly more background discussion, discussion of the modelling approach, and results, the reader is referred to [!citet](sheldon2021).
+
+## Model setup
+
+A single-well ATES system with radial symmetry is explored here, in which hot water is injected and produced from the same well.  Hence, this may be simulated using "RZ" coordinates:
+
+!listing modules/porous_flow/examples/ates/ates.i block=Problem
+
+!listing modules/porous_flow/examples/ates/ates.i block=GlobalParams
```

Comment:
Do you need to show the GlobalParams block here? Or, if you want all elements from the input file, maybe add a sentence "Common object parameters can be included in the GlobalParams block" or similar.

Reply:
I was scratching my head as to why i included the GlobalParams.   Now i remember - it was to remind the reader that gravity is now acting in the "y" direction.   I've included a sentence along those lines.